workspaces/ui-v2/config/webpack.config.js

```diff
@@ -591,6 +595,24 @@ module.exports = function (webpackEnv) {
         )
       ),
       new OpticWasmWebpackPlugin(),
+      shouldUploadSentrySourcemaps &&
```

Comment:
This should only run when we are actually distributing the published versions (since bundles are generated with hashes that can change)